DemoApp/BannerShowingConnectionDelegate.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+final class BannerShowingConnectionDelegate {
+    // MARK: - Private Properties
+    
+    private let navigationController: UINavigationController
+    private var connectionEstablishmentTime: Date?
```

Comment:
This property improves UX by allowing us to ignore momentary states changes, plus Vojta mentioned that ConnectionController might fire several consecutive connection/disconnection events. 
So this way we can throttle banner appearance